.github/workflows/build-docker.yaml

```
@@ -4,6 +4,11 @@ name: Build Docker
 on:
   workflow_call:
     inputs:
+      branch_name:
+        description: Tag used in the docker image (branch/tag/commit).
+        required: true
+        type: string
+        default: ros2-devel
```

actionlint (1.7.4)

input "branch_name" of workflow_call event has the default value "ros2-devel", but it is also required. if an input is marked as required, its default value will never be used

(events)